scalafix-core/src/main/scala/scalafix/v1/Symbol.scala

```diff
@@ -9,7 +9,7 @@ final class Symbol private (val value: String) {
   def isGlobal: Boolean = value.isGlobal
   def isLocal: Boolean = value.isLocal
   def owner: Symbol = Symbol(value.owner)
-  def info(doc: SemanticDoc): SymbolInfo = doc.info(this)
+  def info(doc: SemanticDoc): Option[SymbolInfo] = doc.info(this)
```

Comment:
Looks like there are two ways of doing the same thing: `doc.info` and `symbol.info`. How about we keep only the former?